apps/console/src/features/users/components/guests/pages/guest-users-list.tsx

```diff
@@ -337,7 +335,15 @@ export const GuestUsersList: FunctionComponent<GuestUsersListInterface> = (
                 render: (invite: UserInviteInterface) => {
                     const status: string = invite.status;
 
-                    return status.charAt(0).toUpperCase() + status.substr(1).toLowerCase();
+                    return (
+                        <Header as="h6" data-componentid={ `${ componentId }-invite-status` }>
```

Comment:
This component is common to both admin inviting flow and sub org invite flow. Can we check if this element affect the admin flow?

Reply:
Ack will check this.